GETTING HELP.md

```diff
@@ -0,0 +1,14 @@
+Getting help with Etcher
+========================
+
+There are various ways you can get support for Etcher if you're having any issue, concern or idea you'd like to share with us.
```

Comment:
Nice, thanks, I'm actually looking for this kind of style comments, so not considered rude at all :)